src/main/java/frc/robot/superstructure/Superstructure.java

```diff
@@ -55,73 +57,81 @@ public Command setState(RobotState state) {
-			funnelStateHandler.setState(FunnelState.SPEAKER),
-			swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE.withAimAssist(AimAssist.SPEAKER))
+				swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE.withAimAssist(AimAssist.SPEAKER)),
+				flywheel.getCommandsBuilder().setState(FlywheelState.SHOOTING),
+				funnelStateHandler.setState(FunnelState.SPEAKER)
 		);
 	}
 
 	public Command preAmp() {
 		return new ParallelCommandGroup(
-			swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE.withAimAssist(AimAssist.AMP)),
-			funnelStateHandler.setState(FunnelState.STOP)
+				swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE.withAimAssist(AimAssist.AMP)),
+				flywheel.getCommandsBuilder().setState(FlywheelState.DEFAULT),
+				funnelStateHandler.setState(FunnelState.STOP)
 		);
 	}
 
 	public Command amp() {
 		return new ParallelCommandGroup(
-			swerve.getCommandsBuilder()
-				.saveState(SwerveState.DEFAULT_DRIVE.withAimAssist(AimAssist.AMP))
-				.until(() -> swerve.isAtHeading(Field.getAngleToAmp())),
-			funnelStateHandler.setState(FunnelState.AMP)
+				swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE.withAimAssist(AimAssist.AMP)),
+				flywheel.getCommandsBuilder().setState(FlywheelState.DEFAULT),
+				funnelStateHandler.setState(FunnelState.AMP)
 		);
 	}
 
 	public Command transferShooterElevator() {
 		return new ParallelCommandGroup(
-			swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE),
-			funnelStateHandler.setState(FunnelState.SHOOTER_TO_ELEVATOR)
+				swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE),
+				flywheel.getCommandsBuilder().setState(FlywheelState.DEFAULT),
+				funnelStateHandler.setState(FunnelState.SHOOTER_TO_ELEVATOR)
 		);
 	}
 
 	public Command transferElevatorShooter() {
 		return new ParallelCommandGroup(
-			swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE),
-			funnelStateHandler.setState(FunnelState.NOTE_TO_SHOOTER)
+				swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE),
+				flywheel.getCommandsBuilder().setState(FlywheelState.DEFAULT),
+				funnelStateHandler.setState(FunnelState.NOTE_TO_SHOOTER)
 		);
 	}
 
 	public Command intakeOuttake() {
 		return new ParallelCommandGroup(
-			swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE),
-			funnelStateHandler.setState(FunnelState.INTAKE_OUTTAKE)
+				swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE),
+				flywheel.getCommandsBuilder().setState(FlywheelState.DEFAULT),
+				funnelStateHandler.setState(FunnelState.INTAKE_OUTTAKE)
 		);
 	}
 
 	public Command shooterOuttake() {
 		return new ParallelCommandGroup(
-			swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE),
-			funnelStateHandler.setState(FunnelState.SHOOTER_OUTTAKE)
+				flywheel.getCommandsBuilder().setState(FlywheelState.SHOOTER_OUTTAKE),
+				swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE),
```

Comment:
make order same as class fields
```suggestion
				swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE),
				flywheel.getCommandsBuilder().setState(FlywheelState.SHOOTER_OUTTAKE),
```